### PR TITLE
 [DO NOT MERGE]Updating documents with ec2-quickstart template changes

### PR DESCRIPTION
**Pull request by SuruchiSaboo**
Updating documents with ec2-quickstart-template changes
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aj-NR** [comment]
This PR can be merged once quickstart PR is merged. Please note prior merger to this PR ,the document on New Relic doc website needs to be updated as this PR has link to it.

**Review by davmayd — APPROVED** [review]

**Pull request merged**
